scripts/zones/Ordelles_Caves/npcs/qm5.lua

```diff
@@ -6,32 +6,25 @@
 -----------------------------------
 package.loaded["scripts/zones/Ordelles_Caves/TextIDs"] = nil;
 -----------------------------------
-require("scripts/globals/settings");
 require("scripts/zones/Ordelles_Caves/TextIDs");
+require("scripts/globals/npc_util");
 -----------------------------------
 
 function onTrade(player,npc,trade)
-
-    if (player:getVar("darkPuppetCS") >= 3 and player:hasItem(16940) == false) then
-        if (trade:hasItemQty(16681,1) and trade:getItemCount() == 1) then -- Trade Gerwitz's Axe
-            player:tradeComplete();
-            player:messageSpecial(GERWITZS_SWORD_DIALOG);
-            SpawnMob(17568136):updateClaim(player);
-        end
+    -- DARK PUPPET: Gerwitz's Axe
+    if ( player:getVar("darkPuppetCS") >= 3 and not player:hasItem(16940) and npcUtil.tradeHas(trade, 16681) ) then
```

Comment:
your formatting went weird here.

Reply:
I put the extra space padding inside the parens sometimes for conditionals that end with multiple closing parents, so it's easier to see what each closing paren is closing.